.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,408 @@
+name: "Continuous Integration"
```

Comment:
Since this is a library we're interested in maintaining a BC API on, lets add this step too please?

```
  roave-backwards-compatibility-check:
    name: "Check for Backward Compatibility breaks"
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
      - name: "Install PHP"
        uses: shivammathur/setup-php@v2
        with:
          coverage: "none"
          php-version: "7.4" # https://github.com/Roave/BackwardCompatibilityCheck/issues/283
      - name: "Require Roave/BackwardCompatibilityCheck"
        run: "composer require --no-update --no-interaction --prefer-dist --prefer-stable --dev roave/backward-compatibility-check:6.0.x-dev"
      - name: "Composer update with new requirements"
        run: "composer update --no-interaction --prefer-dist --prefer-stable"
      - name: "Check for BC breaks"
        run: "vendor/bin/roave-backward-compatibility-check --format=markdown"
```

If it fails without dev dependencies, you can add the `--install-development-dependencies` to `vendor/bin/roave-backward-compatibility-check`